README.rst

```diff
@@ -5,7 +5,7 @@ Wildfish Django Starter
     :alt: Build Status
     :target: https://travis-ci.org/wildfish/wildfish-django-starter
     
-A Django 2.0 friendly project cookiecutter template to kick start development for new projects.  Includes apps and settings we use in the majority of projects, along with an integrated version of our other cookiecutter-django-crud template which will also generate a model, CRUD views and tests.
+A Django 2.2 friendly project cookiecutter template to kick start development for new projects.  Includes apps and settings we use in the majority of projects, along with an integrated version of our other cookiecutter-django-crud template which will also generate a model, CRUD views and tests.
```

Comment:
I think let's just take out the version so we don't forget to update it. Change to:

"A Django project cookiecutter"

Reply:
Yep, added.